Controller/HelperController.php

```diff
@@ -333,8 +333,7 @@ public function retrieveAutocompleteItemsAction(Request $request)
             throw new AccessDeniedException();
         }
 
-        // subject will be empty to avoid unnecessary database requests and keep autocomplete function fast
```

Comment:
Yes there is a reason why it was done this way (performances). But the current way break a documented feature (dynamic edit form based on subject).